test/e2e_node/topology_manager_test.go

```diff
@@ -170,9 +170,7 @@ func findNUMANodeWithoutSRIOVDevicesFromSysfs(numaNodes int) (int, bool) {
 	}
 
 	if len(pciPerNuma) == 0 {
-		// if we got this far we already passed a rough check that SRIOV devices
-		// are available in the box, so something is seriously wrong
-		framework.Failf("failed to find any VF devices from %v", pciDevs)
```

Comment:
It might be good to have some kind of log here to help us know that we hit this condition.

Reply:
Done.